vignettes/predict-example.Rmd

```diff
@@ -187,13 +187,13 @@ For each unit $j$ in our dataset, for each value of $x_3$ in the bottom/top quar
 #   xPredQ1[,2] <- x20[j,2]
 #   xPredQ4[,2] <- x20[j,2]
 #   # predict the treatment effect for each simulated unit
-#   tauPredQ1 = predict(bcfObj              = bcf_out,
+#   tauPredQ1 = predict(object              = bcf_out,
```

Comment:
This is another one I'm wonder why it got renamed.

Reply:
I don't remember the right words but i think this is what i want to say: "You are *overloading the method predict, therefore you need to have the same arguments with the same order as the original method"